### ISSUE TITLE
Wrong occlusion score 

### ISSUE THREAD

**Issue by oscarfossey** [pixelite1201/agora_evaluation#25]
Hello thanks for the great work!

I noticed inconsistent occlusion score for many samples. I get an occlusion score of 99.41 for the green character which is in front of the camera. I use the smplx dataframe. Do you have the correct occlusion scores somewhere? 

sample_id : ag_trainset_renderpeople_bfh_brushifygrasslands_5_15_00561.png

![image](https://github.com/pixelite1201/agora_evaluation/assets/97680182/e810ccc8-9632-42f6-9bf7-56a94867dce6)

**Comment by fabienbaradel** [comment]
Up @oscarfossey @pixelite1201 do you have any idea if this error is fixed?
How can we re-compute the occlusion for the validation set?
Thanks for your help,

**Comment by pixelite1201** [comment]
Certainly! Here's a revised version of your message with improved clarity and structure:

Hello,

We calculate the occlusion score using segmentation masks. For each image, we have a full segmentation mask where each person is represented with a different color. Additionally, we have individual segmentation masks for each person without any occlusion.

To compute the occlusion percentage, we compare the pixel count of a person in the full segmentation mask with the pixel count in their individual segmentation mask. If Unreal outputs an incorrect segmentation mask for a particular individual, it could lead to an incorrect occlusion score.

Unfortunately, the original code for this calculation is no longer available. If anyone is able to reimplement it, please consider making a pull request.

Thank you
